i18n/ja/docusaurus-plugin-content-docs/current/tutorials/additive-loading-scenes.md

```diff
@@ -7,6 +7,8 @@ thumb: "https://s3-eu-west-1.amazonaws.com/images.playcanvas.com/projects/12/685
 
 シーンの読み込みに関する詳細なドキュメントは [ユーザーマニュアル][documentation-page] にまとめられています。
 
-<iframe loading="lazy" src="https://playcanv.as/e/p/cjBInud1/" title="Additive Loading Scenes"></iframe>
+<div className="iframe-container">
+    <iframe loading="lazy" src="https://playcanv.as/e/p/cjBInud1/" title="Additive Loading Scenes"></iframe>
+</div>
```

Comment:
Japanese MD files come from Transifex - they shouldn't be edited directly. Best to revert all MD changes in the `i18n` folder.

Reply:
Ahh ok. Will remove.

Reply:
Ok done